src/main/java/Duke.java

```diff
@@ -1,10 +1,291 @@
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import src.main.java.Task;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+
+    private static ArrayList<String> stringList = new ArrayList<>();
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    private static ArrayList<String> dueDate = new ArrayList<>();
+
+    private static int listCount = 0;
+
+    private static final String DATA_FILE_PATH = "./data/data.txt";
+
+    private static final String LINE = "-------------------------------------------------------------------\n";
+    private static final String GREET_USER = "Hello! I'm Duke \nWhat can I do for you?";
+    private static final String EXIT_MESSAGE = "Bye. Hope to see you again soon!";
+
+    private static void greetUser() {
+        System.out.println(GREET_USER + "\n \n" + LINE);
+    }
+
+    private static void markAsDone(String line) {
+        String taskNumber = line.substring(5);
+        int taskListElement = Integer.parseInt(taskNumber) - 1;
+        taskList.get(taskListElement).markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("[X] " + taskList.get(taskListElement).getDescription());
+        System.out.println(LINE);
+    }
+
+    private static void list() {
```

Comment:
Perhaps this name could be more descriptive? It may be unclear whether you are printing a list, or it is the noun list, so you could [avoid misleading names](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-misleading-names).